run_cgw.sh

```diff
@@ -17,37 +17,43 @@ DEFAULT_DB_PORT=5432
 DEFAULT_REDIS_DB_IP="127.0.0.1"
 DEFAULT_REDIS_DB_PORT=6379
 
+LOG_LEVEL="${CGW_LOG_LEVEL:-info}"
 ID="${CGW_ID:-1}"
-
-GRPC_IP="${CGW_GRPC_IP:-$DEFAULT_SRC_IP}"
-GRPC_PORT="${CGW_GRPC_PORT:-$DEFAULT_GRPC_PORT}"
 WSS_LOCAL_IP="${CGW_WSS_IP:-$DEFAULT_WSS_IP}"
 WSS_LOCAL_PORT="${CGW_WSS_PORT:-$DEFAULT_WSS_PORT}"
+GRPC_IP="${CGW_GRPC_IP:-$DEFAULT_SRC_IP}"
+GRPC_PORT="${CGW_GRPC_PORT:-$DEFAULT_GRPC_PORT}"
 KAFKA_IP="${CGW_KAFKA_IP:-$DEFAULT_KAFKA_IP}"
 KAFKA_PORT="${CGW_KAFKA_PORT:-$DEFAULT_KAFKA_PORT}"
 DB_IP="${CGW_DB_IP:-$DEFAULT_DB_IP}"
 DB_PORT="${CGW_DB_PORT:-$DEFAULT_DB_PORT}"
-DB_USR="${CGW_DB_USER:-cgw}"
-DB_PASS="${CGW_DB_PASS:-123}"
+DB_USERNAME="${CGW_DB_USER:-cgw}"
+DB_PASSWORD="${CGW_DB_PASS:-123}"
 REDIS_DB_IP="${CGW_REDIS_DB_IP:-$DEFAULT_REDIS_DB_IP}"
 REDIS_DB_PORT="${CGW_REDIS_DB_PORT:-$DEFAULT_REDIS_DB_PORT}"
```

Comment:
It looks like we're not using any authentication on REDIS today.   We should add a Jira ticket to track adding this, as it'll be required from a security perspective.  All external services need to use some form of authentication

Reply:
Good catch; i've added a new jira issue to track this, thanks;